### ISSUE TITLE
[php7] stripcslashes causes zephir build to fail when used within another function

### ISSUE THREAD

**Issue by Izopi4a** [zephir-lang/zephir#1263]
test.zep

```
namespace Test;

class Test {

    public function koko(value) {

        let value = str_replace(["\\", "\"", "'"], "", value);

        let value = filter_var(value, FILTER_SANITIZE_STRING);
        return trim(stripcslashes(strip_tags(value)));
    }
}


```

compile-errors.log

```
/home/izopi4a/test_zephir/test/ext/test/test.zep.c: In function ‘zim_Test_Test_koko’:
/home/izopi4a/test_zephir/test/ext/test/test.zep.c:72:2: error: incompatible type for argument 1 of ‘zephir_stripcslashes’
  zephir_stripcslashes(_0, &_4 TSRMLS_CC);
  ^
In file included from /home/izopi4a/test_zephir/test/ext/test/test.zep.c:16:0:
./kernel/string.h:82:6: note: expected ‘struct zval *’ but argument is of type ‘zval’
 void zephir_stripcslashes(zval *return_value, zval *str);
      ^
make: *** [test/test.lo] Error 1
/home/izopi4a/test_zephir/test/ext/test/test.zep.c: In function ‘zim_Test_Test_koko’:
/home/izopi4a/test_zephir/test/ext/test/test.zep.c:72:2: error: incompatible type for argument 1 of ‘zephir_stripcslashes’
  zephir_stripcslashes(_0, &_4 TSRMLS_CC);
  ^
In file included from /home/izopi4a/test_zephir/test/ext/test/test.zep.c:16:0:
./kernel/string.h:82:6: note: expected ‘struct zval *’ but argument is of type ‘zval’
 void zephir_stripcslashes(zval *return_value, zval *str);
      ^
make: *** [test/test.lo] Error 1


```

**Comment by Izopi4a** [comment]
@ovr

**Comment by kuzmichus** [comment]
+1

**Comment by sjinks** [comment]
Fixed in efeaa0ee6c1bd005fe3d175e83444b7ec93603da
